boudicca.base/common-model/src/main/kotlin/base/boudicca/format/ListFormat.kt

```diff
@@ -8,7 +8,10 @@ package base.boudicca.format
  * all methods may throw exceptions on wrong formatted values
  */
 object ListFormat {
-    fun parseFromString(value: String): List<String> {
```

Comment:
Why does the parameter need to be nullable here? when does it make sense to parse a null value?